searchlib/src/vespa/searchlib/diskindex/disktermblueprint.h

```diff
@@ -26,6 +26,8 @@ class DiskTermBlueprint : public queryeval::SimpleLeafBlueprint
     mutable std::shared_ptr<BitVector> _late_bitvector;
 
     const BitVector* get_bitvector() const;
+    void log_bitvector_read() const;
+    void log_posting_list_read() const;
```

Comment:
Maybe these should be tagged as `__attribute__((noinline))` to keep them off the hot path?